pyproject.toml

```diff
@@ -22,7 +22,7 @@ classifiers = [
 dynamic = ["version"]
 dependencies = [
     "attrs",
-    "pyproj~=3.1",
+    "pyproj~=3.7",
```

Comment:
@marklit  what do you think about `pyproj>=3.1,<4.0` ?

Reply:
Sounds good to me.